coredis/recipes/locks/lua_lock.py

```diff
@@ -128,7 +128,9 @@ async def __aenter__(
 
         # force blocking, as otherwise the user would have to check whether
```

Comment:
This comment is clearly not clear :smile: (copy paste error from https://github.com/NoneGG/aredis/blob/master/aredis/lock.py#L88)